coral/src/domain/connector/connector-api.ts

```diff
@@ -135,7 +135,10 @@ const createConnectorRequest = (
   return api.post<
```

Comment:
This way, the `requestOperationType` still needs to be passed in `ConnectorRequest` when mutating (there's a type error right now). I would add a custom type for the params that we need to path, I think we done it similar in the apis for approving / declining requests.

Reply:
Ah, thank you. I think `Omit<KlawApiRequest<"createConnectorRequest">, "requestOperationType">` would work in this case?

Reply:
Yes, I think that's a good solution for that.